src/docs/user/references/DataConnectors.adoc

```diff
@@ -216,3 +216,78 @@ Currently, we have connectors for MongoDB and Cassandra.
 The task requires the following third-party credentials: `<NoSQL_NAME>_USERNAME` and `<NoSQL_NAME>_PASSWORD`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
 The imported data is exported in a JSON format using the variable `DATAFRAME_JSON`.
 
+=== Cloud
+
+Cloud data connectors allow to interact with cloud storage services. Currently we provide support for Amazon S3, Azure Storage and Azure Data Lake.
+
+==== Azure Data Lake
+
+The Azure Data Lake connector allows to upload U-SQL scripts and then execute them as Data Lake Analytics jobs. It requires an existing Data Lake Analytics account and its corresponding Data Lake Store account. The workflow consists of three tasks:
```

Comment:
Maybe:
The connector workflow consists of ...

Reply:
good catch! I'll wait for any comment from Amine to push a new commit with this correction :)